app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+
+  after_save :upadate_likes_counter
+
+  def upadate_likes_counter
+    post.increment!(:likes_counter)
+  end

```

Comment:
[OPTIONAL] After using the rails active callbacks after_save method, kindly consider making the update_likes_counter method private to make your code simple and up to standard.